server/v1/controllers/Parties.js

```diff
@@ -0,0 +1,44 @@
+const allParties = [
+    {
+        id: 1,
+        name: 'Action Peoples Congress',
+        hqAddress: 'Wuse Zone II, Abuja, Nigeria',
+        logoUrl: '../assets/images/apc.png'
+    },
+    {
+        id: 2,
+        name: 'All Democratic Congress',
+        hqAddress: 'Wuse Zone I, Abuja, Nigeria',
+        logoUrl: '../assets/images/adc.png'
+    },
+    {
+        id: 3,
+        name: 'Alliance for Democracy Party',
+        hqAddress: 'Gwarinpa, Abuja, Nigeria',
+        logoUrl: '../assets/images/adp.jpg'
+    },
+    {
+        id: 4,
+        name: 'All Peoples Grand Alliance',
+        hqAddress: 'Garki, Abuja, Nigeria',
+        logoUrl: '../assets/images/adc.png'
+    }
+];
+
+const Parties = {
+    viewParties(req, res) {
+        if (req.userData === 1) {      
```

Comment:
Why did you hardcode 1 into this code. Since this endpoint is to return all political parties, there is no point making a check. Just return the parties list

Reply:
Since there is no need to check the end point, I've decided to remove that part of the code. I having issues with **commit squash** though.